lms/templates/problem.html

```diff
@@ -56,6 +56,11 @@ <h2 class="problem-header">
     % if check_button:
     <input class="check ${ check_button }" type="button" data-checking="${ check_button_checking }" value="${ check_button }" />
     % endif
+    % if exceeded_time_limit:
+    <p class="exceeded-time-limit-message">
```

Comment:
Ah I see, sorry I didn't see this comment until now. I didn't want to confuse the semantics of notification message vs button, so I opted for the message instead. 

I've since deleted the local copy of this branch, so it's hard to revise. After merging into the upcoming RC, I'll PR another revision on this issue.